### PR TITLE
Feature/fix old issues

### PR DESCRIPTION
**Pull request by ryanrampage1**
- new package name
- retrolambda
- various lint fixes

### CODE REVIEW

File: app/src/main/java/com/spartahack/spartahack17/Adapters/EventListAdapter.java [hunk 1]
```diff
@@ -43,7 +44,7 @@ public void remove(int position){
         @Bind(R.id.title) TextView title;
         @Bind(R.id.time) TextView time;
         @Bind(R.id.description) TextView description;
-        @Bind(R.id.location) TextView locaiton;
+        @Bind(R.id.location) TextView location;
```
Comment (mhammerly) [inline app/src/main/java/com/spartahack/spartahack17/Adapters/EventListAdapter.java:47]: lmao

Comment (bogderp) [reply app/src/main/java/com/spartahack/spartahack17/Adapters/EventListAdapter.java:47]: quality review @mhammerly ;)

**Comment by mhammerly** [comment]
LGTM, provided it is okay that [this line](https://github.com/SpartaHack/SpartaHack-Android/pull/66/commits/efad52ae61cca63b698f7f90be202cbd004fb3a5?diff=unified#diff-2c8eaed4f323a07d81a7da43d1be4bb5L47) and some other `layout_*` attributes are removed without replacement

**Comment by ryanrampage1** [comment]
layout_\* lines were removed due to lint errors. they were at one point in a relative layout so they were needed, now in a linear layout they do not do anythin g

**Pull request merged**